CHANGES.rst

```diff
@@ -1,6 +1,13 @@
 Change Log
 ==========
 
+**Unreleased**
```

Comment:
The formatting here should be that of a second-level header, similar to the below that says `6.4.0 (2019/09/21)`. So, it should be

    Unreleased
    ----------

Then, below that, there should be a bolded section that says `Added`:

    **Added**